scripts/plugins/storage/postgres/init.sql

```diff
@@ -807,36 +807,44 @@ INSERT INTO foglamp.configuration ( key, description, value )
               ' { "plugin" : { "type" : "string", "value" : "omf", "default" : "omf", "description" : "Module that OMF North Plugin will load" } } '
             );
 
--- SEND_PR_1 - OMF Translator for readings
+-- North Readings to PI - OMF Translator for readings
 INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'SEND_PR_1',
+     VALUES ( 'North Readings to PI',
```

Comment:
fix for FOGL-1565 should go in first to make sure, sending process is working in develop!

Reply:
yes, post FOGL-1565 aka #940 merger, we will need to add upgrade/downgrade scripts.